docs/src/development/variables.md

```diff
@@ -2,115 +2,160 @@
 
 ```bash
-$ platform variable:create -l environment -e master --prefix env: --name NODE_ENV --value production --visible-build true --inheritable false
+$ platform variable:create -l environment -e main --prefix env: --name NODE_ENV --value production --visible-build true --inheritable false
 $ platform variable:create -l environment -e staging --prefix env: --name NODE_ENV --value development --visible-build true --inheritable true
 ```
 
-With these two commands, while `NODE_ENV` will be `production` on the default branch, it will be `development` on staging as well as each of its child environments. Note that build visible environment variables change the application's build configuration ID - value updates will trigger a rebuild of the application in the same way that a commit would. 
+Now `NODE_ENV` is `production` on the default branch but `development` on staging and each of its child environments. Note that build visible environment variables change the application's build configuration ID: value updates will trigger a rebuild of the application in the same way that a commit would. 
 
-### Platform.sh-provided variables
+### Use Platform.sh-provided variables
 
-Platform.sh also provides a series of variables by default.  These inform an application about its runtime configuration.  The most important of these is relationship information, which tells the application how to connect to databases and other services defined in `services.yaml`.  They are always prefixed with `PLATFORM_*` to differentiate them from user-provided values.
+Platform.sh also provides a series of variables by default that inform an application about its runtime configuration. They are always prefixed with `PLATFORM_*` to differentiate them from user-provided values and you can't set or update them directly.
 
-#### Variables available only at build time
+The most important of these variables is relationship information, which tells the application how to connect to databases and other services defined in `services.yaml`. 
 
-The following variables are only available at build time, and may be used in a build hook:
+The following table presents the available variables and whether they are available during builds and at runtime.
 
-* **PLATFORM_OUTPUT_DIR**: The output directory for compiled languages at build time. Will be equivalent to `PLATFORM_APP_DIR` in most cases.
+| Variable name             | Build | Runtime | Description |
+| ------------------------- | ----- | ------- | ----------- |
+| PLATFORM_OUTPUT_DIR       | Yes   | No      | The output directory for compiled languages at build time. Equivalent to `PLATFORM_APP_DIR` in most cases. |
```

Comment:
```suggestion
| `PLATFORM_OUTPUT_DIR`       | Yes   | No      | The output directory for compiled languages at build time. Equivalent to `PLATFORM_APP_DIR` in most cases. |
```

Reply:
Worthwhile doing this all the way down?

Reply:
I had it that way initially, but then I noticed that when it was that way, the words would wrap. That made it much harder to read the names. Without the code markup, they didn't wrap and took up enough space to read them. 
![wrap with code](https://user-images.githubusercontent.com/26428882/132864353-73f34972-2e77-4750-95f9-2908611355d4.png)
vs
![unwrap without code](https://user-images.githubusercontent.com/26428882/132864449-2cfddc57-e115-4c9a-b88f-17954b92b5ad.png)
So if you like, I can change it that way, but it seemed harder to use.